Classes/Backend/TceMain.php

```diff
@@ -262,6 +268,16 @@ public function processCmdmap_postProcess(&$command, $table, $id, &$relativeTo,
             if (!empty($clipboardCommand['paste']) && strpos($clipboardCommand['paste'], 'tt_content|') === 0) {
                 $properties = (array) $clipboardCommand['update'];
                 $clipboardCommand = GeneralUtility::trimExplode('|', $clipboardCommand['paste']);
+            } elseif ($pasteUpdate !== false) {
+                $copyCommand = $this->getCopyPasteCommand();
+                $clipboardCommand = $copyCommand[$table][abs($id)];
+                if ((integer) $pasteDataMap[$table][$reference->copyMappingArray[$table][$id]]['colPos'] > ContentService::COLPOS_FLUXCONTENT) {
+                    // remove colPos from pasteDataMap if greater than 18181
+                    // to avoid merging the colPos after finished the processCmdmap_postProcess
+                    // in the DataHandler
+                    // if colPos < 18181 then the target is outside of a flux:grid
+                    unset ($pasteDataMap[$table][$reference->copyMappingArray[$table][$id]]['colPos']);
```

Comment:
I'm still not sure we need this... but if we do: setting colPos to COLPOS_FLUXCONTENT (after we've caught and handled the `> COLPOS_FLUXCONTENT` case!) should be the right way to do it.

The change here makes sense as a separate pull request; assuming the problem isn't that our logic in ContentService is doing something wrong when pasting records?